api/routes/validation-test.ts

```diff
@@ -0,0 +1,61 @@
+import { Type } from '@sinclair/typebox';
+
+import type { FastifyInstanceWithTypeProvider } from '..';
+import { responseSchema, subSchema } from '../schemas/example';
+
+export const testValidatedRoutes = (
+  fastify: FastifyInstanceWithTypeProvider,
+  _options: never,
+  done: (err?: Error) => void
+): void => {
+  fastify.get(
+    '/route-with-validation',
+    {
+      schema: {
+        querystring: Type.Object({
+          foo: Type.Number(),
+          bar: Type.String()
+        })
+      }
+    },
```

Comment:
Is it possible to ESLint warn/error on routes without explicit response schemas? Or, schemas in general?

Reply:
Not yet, no: https://github.com/fastify/fastify/issues/4434

For now how about a simple route checklist?

- [ ] Has tests
- [ ] Has a request schema
- [ ] Has a reply schema

Reply:
The checklist looks solid and sets up a baseline on what to check for while reviewing PRs